video/out/vo_sdl.c

```diff
@@ -599,7 +599,7 @@ static void wait_events(struct vo *vo, int64_t until_time_us)
             break;
         }
         case SDL_MOUSEMOTION:
-            mp_input_set_mouse_pos(vo->input_ctx, ev.motion.x, ev.motion.y);
+            mp_input_set_mouse_pos(vo->input_ctx, ev.motion.x, ev.motion.y, 0);
```

Comment:
SDL should be able to handle modifiers.

Reply:
As far as I can see, the SDL vo does not handle keyboard modifiers at all with mouse events at the moment. 
The SDL events also do not contain any field with the modifiers held, should I query the modifier state in all these events?

Reply:
Does `SDL_GetModState()` not work?